parlai/mturk/tasks/turn_annotations/constants.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+AGENT_0 = 'Person1'
+AGENT_1 = 'Person2'
+
+WAITING_MSG = 'Please wait while we match you with another worker...'
+
+ANNOTATIONS_CONFIG = [
+    {
+        'value': 'bucket_0',
+        'name': 'Bucket 0',
+        'description': 'this response implies something...0',
+    },
+    {
+        'value': 'bucket_1',
+        'name': 'Bucket 1',
+        'description': 'this response implies something...1',
+    },
+    {
+        'value': 'bucket_2',
+        'name': 'Bucket 2',
+        'description': 'this response implies something...2',
+    },
+    {
+        'value': 'bucket_3',
+        'name': 'Bucket 3',
+        'description': 'this response implies something...3',
+    },
+    {
+        'value': 'bucket_4',
+        'name': 'Bucket 4',
+        'description': 'this response implies something...4',
+    },
+]
+
+ONBOARD_SUBMIT = '[ONBOARD_SUBMIT]'
+ONBOARD_TRY_AGAIN = '[ONBOARD_TRY_AGAIN]'
+ONBOARD_FAIL = '[ONBOARD_FAIL]'
+ONBOARD_SUCCESS = '[ONBOARD_SUCCESS]'
+
+ONBOARD_TASK_DATA = [
```

Comment:
this coudl use some explanation

Perhaps instead of lorems we could do

"Quality utterance #1, where the user must check bucket_0"

Reply:
Putting: "Model utterance where user should select bucket_0."